compiler/rustc_infer/src/infer/relate/nll.rs

```diff
@@ -262,7 +262,7 @@ where
     }
 
     #[instrument(skip(self), level = "debug")]
-    fn instantiate_binder_with_placeholders<T>(&mut self, binder: ty::Binder<'tcx, T>) -> T
+    fn enter_forall_and_leak_universe<T>(&mut self, binder: ty::Binder<'tcx, T>) -> T
```

Comment:
can this function be inlined into `enter_forall`? afaict we only ever use `enter_forall` directly